cymetric/metrics.py

```diff
@@ -467,3 +468,45 @@ def timelist(info):
 
 del _tldeps, _tlschema
 
+# Quantity per GigaWattElectric (integration of total power) in Inventory [kg/GWe]
+_invdeps = ['ExplicitInventory','TimeSeriesPower']
+
+_invschema = [
+    ('SimId', ts.UUID),
+    ('AgentId', ts.INT),
+    ('Time', ts.INT),
+    ('InventoryName', ts.STRING),
+    ('NucId', ts.INT),
+    ('Quantity', ts.DOUBLE)
+    ]
+@metric(name='InventoryQuantityPerTotalGWe', depends=_invdeps, schema=_invschema)
+def inventory_quantity_per_total_gwe(expinv,power):
+    """Inventory Quantity per GWe metric returns the explicit inventory table with quantity
+    in units of kg/GWe, calculated by dividing the original quantity by the integration of total 
+    electricity generated in TimeSeriesPower metric from time 0 to time of the simulation.
+    """
+    power = pd.DataFrame(data={'SimId': power.SimId,
+			      'AgentId': power.AgentId,
+                              'Time': power.Time,
+                              'Value': power.Value},
+			columns=['SimId','AgentID','Time', 'Value'])
+    power_index = ['SimId','Time']
+    power = power.groupby(power_index).sum()
+    df1 = power.reset_index()
+    total_power = 0
+    for t in range(len(df1)):   
+        realpower = total_power + df1.Value[t]
+        total_power += realpower
```

Comment:
total_power += df1.Value[t]
```suggestion
        total_power += realpower
```